### PR TITLE
dynamic inputs changes

### PR DESCRIPTION
**Pull request by vladsavchuk**

### CODE REVIEW

File: osafw-app/App_Data/template/common/form/showform/select.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<select id="<~field>" name="item[<~field>]" class="form-select <~class_control>" <~attrs_control noescape>>
+<select id="<~field>" name="item[<~field>]" class="form-select <~class_control>" <~/common/required if="required"> <~attrs_control noescape>>
```
Comment (osalabs) [inline osafw-app/App_Data/template/common/form/showform/select.html:1]: are you sure required is here? required class only used on labels, not on form controls itself

File: osafw-app/App_Data/template/common/icons/alarm-fill.html [hunk 2]
```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-alarm-fill" viewBox="0 0 16 16">
+<svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" fill="currentColor" class="bi bi-alarm-fill" viewBox="0 0 16 16">
```
Comment (osalabs) [inline osafw-app/App_Data/template/common/icons/alarm-fill.html:1]: well, "16" used in bs5 icons original svg code, if we going to use "1em", then any updates (new icons) need to apply this change too

File: osafw-app/App_Data/template/common/form/showform/select.html (hunk 1 above)
Comment (vladsavchuk) [reply osafw-app/App_Data/template/common/form/showform/select.html:1]: yes, we have required for other inputs (input, email, number) and textarea:
https://github.com/osalabs/osafw-asp.net-core/blob/master/osafw-app/App_Data/template/common/form/showform/input.html

In other way the browser will not prevent empty values submission.

File: osafw-app/App_Data/template/common/icons/alarm-fill.html (hunk 2 above)
Comment (vladsavchuk) [reply osafw-app/App_Data/template/common/icons/alarm-fill.html]: Okay, I reverted that commit and added a CSS rules to the `site.css`. As BS icons are always squared it'll work. In other SVG icons fonts/sets there could be rectangular shapes and in that case every icon should have it's own width/height proportion according to the view box, like:

`width="0.875em" height="1em" viewBox="0 0 448 512"`

File: osafw-app/App_Data/template/common/form/showform/select.html (hunk 1 above)
Comment (osalabs) [reply osafw-app/App_Data/template/common/form/showform/select.html:1]: ok, I remembered it - it's required attribute, not a class

**Pull request merged**